src/js/legacy/mega-menu.js

```diff
@@ -50,8 +90,9 @@ function initNavMenu() {
   const menuElement = document.querySelector('#vetnav');
   const openMenuElement = document.querySelector('.vetnav-controller-open');
   const closeMenuElement = document.querySelector('.vetnav-controller-close');
+  const menuElements = document.querySelectorAll('#vetnav-menu li');
```

Comment:
Instead of adding this here, you could query for it in `menuElement` in the constructor for MegaMenu.

Reply:
Thanks @jbalboni, sounds good, changed.